README.rst

```diff
@@ -23,6 +23,10 @@ Please do not report security issues in public. Please email security@appsembler
 
 Getting Help
 ------------
+There are two docs about the Trinity project:
+
+ - [The Trinity Backend](https://github.com/appsembler/openedx-docs/blob/master/openedx/Trinity/OAuth2.0-integration.md)
```

Comment:
Remove the indentation as this shows as a code block for these two bullets

Reply:
@OmarIthawi Also, these links go to a private repository, you might want to either move or copy them to this repo.

But we should really create an open developer docs repo for the docs we can share with the public (totally outside of the scope of this PR, of course)

Reply:
I like the idea of moving the docs here! Thanks @johnbaldwin!